tests/check/test_sema_check_cli.py

```
@@ -0,0 +1,25 @@
+import sys
+import logging
+from pathlib import Path
+from sema.check.__main__ import main as query_main
+
+log = logging.getLogger(__name__)
+
+
+def test_main():
+    log.info(f"test_main_check")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)